app/timetables/tests/test_models.py

```diff
@@ -209,3 +209,54 @@ def test_duplicate_dish_name_cannot_be_saved(self):
         dish = Dish(name='eba')
 
         self.assertRaises(IntegrityError, dish.save)
+
+
+class MenuItemTest(TestCase):
+    """Tests the MenuItem model."""
+
+    def setUp(self):
+        self.timetable_object = Timetable.objects.create(
+            name='timtable-item',
+            code='FBI23212',
+            api_key='419223',
+            cycle_length=14,
+            current_cycle_day=2,
+            date_created=datetime.strptime('05 07 2016', '%d %m %Y'),
+            date_modified=datetime.strptime('06 08 2016', '%d %m %Y')
+        )
+
+        self.meal_object = Meal.objects.create(
+            name='breakfast',
+            start_time=datetime.time(datetime.strptime('5:7:9', '%H:%M:%S')),
+            end_time=datetime.time(datetime.strptime('6:7:9', '%H:%M:%S'))
+        )
+
+        self.meal_option = MealOption.objects.create(name='lunch')
+
+        self.menu_item = MenuItem.objects.create(
+            date_created=datetime.strptime('20 07 2016', '%d %m %Y'),
+            date_modified=datetime.strptime('22 08 2016', '%d %m %Y'),
+            cycle_day=4,
+            meal=self.meal_object,
+            meal_option=self.meal_option,
+            timetable=self.timetable_object)
+
```

Comment:
We should be adding a tearDown function to destroy instances we're creating in the setup function.